tests/tacacs/test_ro_user.py

```diff
@@ -49,6 +52,24 @@ def ssh_remote_ban_run(localhost, remote_ip, username, password, cmd):
     logger.info("check command \"{}\" rc={}".format(cmd, res['rc']))
     return res['rc'] != 0 and "Make sure your account has RW permission to current device" in res['stderr']
 
+def wait_for_tacacs(localhost, remote_ip, username, password):
+    current_attempt = 0
+    cmd = 'systemctl status hostcfgd.service'
+    while (True):
+        # Wait for tacacs to finish configuration from hostcfgd
+        logger.info("Check if hostcfgd started and configured tacac attempt = {}".format(current_attempt))
+        time.sleep(SLEEP_TIME)
```

Comment:
Instead of waiting a fix time period, could you wait for the service starting?

Reply:
This was the approach on the previous PR but it doesn't seem to be enough, we need to wait until the service started and tacacs is configured to make it work.
can you please check https://github.com/Azure/sonic-mgmt/pull/3741
Thanks

Reply:
That one is also waiting a fix period.